src/components/FormTodo/FormTodo.tsx

```diff
@@ -0,0 +1,134 @@
+import { useState } from 'react';
+import { Todo, User } from '../../types/types';
+import usersFromServer from '../../api/users';
+import { getNewTodoId } from '../../utils/utils';
+import cn from 'classnames';
+
+type FormTodoProps = {
+  todos: Todo[];
+  addTodo: (todo: Todo) => void;
+};
+
+export const FormTodo: React.FC<FormTodoProps> = ({ todos, addTodo }) => {
+  const [users] = useState<User[]>(usersFromServer);
+
+  const [title, setTitle] = useState('');
+  const [hasTitleErr, setHasTitleErr] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [hasUserIdErr, setHasUserIdErr] = useState(false);
+
+  const handleTitleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(e.target.value);
+    setHasTitleErr(false);
+  };
+
+  const handleUserIdChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+e.target.value);
+    setHasUserIdErr(false);
+  };
+
+  const handleReset = () => {
+    setTitle('');
+    setUserId(0);
+    setHasTitleErr(!title);
+    setHasUserIdErr(!userId);
+  };
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+
+    setHasTitleErr(!title);
+
+    setHasUserIdErr(!userId);
+
+    if (!title || !userId) {
+      return;
+    }
+
+    addTodo({
+      id: getNewTodoId(todos),
+      title,
+      completed: false,
+      userId,
+      user: usersFromServer.find(user => user.id === userId)!,
```

Comment:
fix linter warning.
find returns undefined, if element not found.
```suggestion
      user: usersFromServer.find(user => user.id === userId),
```

Reply:
I found only two ways to fix the ESLint warning if the find method returns undefined: using the ! sign and hardcoding an object. If there is a better option, please let me know.